worldmap.js

```diff
@@ -117,11 +116,95 @@ module.exports = function(RED) {
             }
             node.status({});
         });
-        sockets[this.path].on('connection', callback);
+        sockets[node.path].on('connection', callback);
+    }
+
+    var WorldMap = function(n) {
+        worldMap(this, n);
     }
     RED.nodes.registerType("worldmap",WorldMap);
 
 
+    function HTML(ui, config) {
+        var width = config.width;
+        if (width === 0) {
+            var group = RED.nodes.getNode(config.group);
+            if (group) {
+                width = group.config.width;
+            }
+        }
+        var height = config.height;
+        if (height === 0) {
+            height = 10;
+        }
+        var size = ui.getSizes();
+        var width = (size.sx +size.cx) *width -15;
+        var height = (size.sy +size.cy) *height -15;
```

Comment:
plus I don't think we need to declare vars again.

Reply:
Still needs little fix-up in order to hide scrollbar.